tasks/system_probe.py

```diff
@@ -203,7 +212,63 @@ def test(ctx, skip_object_files=False, only_check_bpf_bytes=False, bundle_ebpf=T
             )
             raise Exit(code=1)
 
-    ctx.run(cmd.format(path=path, go_mod="vendor", bpf_tag=bpf_tag, pkg=pkg))
+    args = {
+        "path": os.environ['PATH'],
+        "bpf_tag": bpf_tag,
+        "output_params": "-c -o " + output_path if output_path else "",
+        "pkgs": packages,
+    }
+
+    ctx.run(cmd.format(**args))
+
+
+@task
+def kitchen_prepare(ctx):
+    """
+    Compile test suite for kitchen
+    """
+    target_packages = []
+    for pkg in TEST_PACKAGES_LIST:
+        target_packages += check_output(
+            "go list -f '{{ .Dir }}' %s" % (pkg),
```

Comment:
Can you use the new [`.format`](https://docs.python.org/3/library/stdtypes.html#str.format) string formatting style? Although the old one is not deprecated `.format` is the preferred way in Python 3.

Reply:
I think in this particular context `.format` doesn't seem to be playing nice with the Go command templating

Reply:
If the problem is the braces you can insert a double braces ([SO](https://stackoverflow.com/a/5466478)) as in
```
"go list -f '{{{{ .Dir }}}}' {}".format(pkg)
```

In any case, since it seems like this syntax is not going away anytime soon let's keep it like this